src/sessions.ts

```diff
@@ -72,4 +72,5 @@ export {
   IsValidWithRoleResponse,
   getSession,
   SessionResponse,
+  ErrorResponse,
```

Comment:
What does this do?

Reply:
I don't remember where the context was but Carolyn asked about having a defined type for the error response(s) so clients could make use of them. by "error response" basically mean any non 20x response from the server.  most common would be 401 and 403, for example.